rotors_control/include/rotors_control/parameters_ros.h

```diff
@@ -12,7 +11,7 @@ template<typename T> inline void GetRosParameter(const ros::NodeHandle& nh,
                                                  const std::string& key,
                                                  const T& default_value,
                                                  T* value) {
-  CHECK_NOTNULL(value);
+  ROS_ERROR_COND(value == nullptr, "value is a nullptr.");
```

Comment:
Or I could, in the CMakeLists, add this: `add_definitions(-DROS_ASSERT_ENABLED)`